get-minikube/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Get minikube cluster
```

Comment:
I think we should move these two actions into a top-level `actions/` directory.

A follow-up PR would probably move the workflows out into a top-level `workflows/` (whatever doesn't get broken into actions).

Reply:
We can also move actions to `.github/actions` directory next to the GitHub Workflow's `.github/workflows` if it make sense

Reply:
About moving actions to top-level dir. We would need to do something like `actions/<action-name>/action.yml` and the same goes if we decide to move it under `.github/actions`. I tried `actions/<action-name>.yml` and it doesn't work.

Reply:
Yeah, I guess you are right, under the top-level directory you will need to add `<action-name>/action.yml`, be it `actions/` or `.github/actions/`